PayPal/src/main/java/com/braintreepayments/api/paypal/vaultedit/PayPalVaultEditAuthRequest.kt

```diff
@@ -1,22 +1,30 @@
 package com.braintreepayments.api.paypal.vaultedit
 
+import com.braintreepayments.api.BrowserSwitchOptions
 import com.braintreepayments.api.core.ExperimentalBetaApi
 
-/**
- * A request used to launch the continuation of the PayPal Edit Vault flow.
- */
 @ExperimentalBetaApi
 sealed class PayPalVaultEditAuthRequest {
 
     /**
-     * The request was successfully created and is ready to be launched by [PayPalLauncher]
+     * The PayPal vault edit flow completed successfully.
+     *
+     * @property riskCorrelationId This ID is used to link subsequent retry attempts if payment is declined
      */
     class ReadyToLaunch internal constructor(
-        internal val requestParams: PayPalVaultEditAuthRequestParams
+        val riskCorrelationId: String,
+        internal var browserSwitchOptions: BrowserSwitchOptions?,
     ) : PayPalVaultEditAuthRequest()
 
     /**
-     * There was an [error] creating the request.
+     * There was an [error] in the PayPal vault edit flow.
      */
-    class Failure internal constructor(val error: Exception) : PayPalVaultEditAuthRequest()
+    class Failure internal constructor(
+        val error: Exception
+    ) : PayPalVaultEditAuthRequest()
+
+    /**
+     * The user canceled the PayPal vault edit flow.
+     */
+    class Cancel internal constructor() : PayPalVaultEditAuthRequest()
```

Comment:
I don't think we need the cancel scenario here - this is returned before any flow is launched right so there is no possibility that user can cancel a this stage?